test/python/providers/fake_provider/test_fake_backends.py

```diff
@@ -57,6 +61,10 @@ def test_fake_backend_v2_noise_model_always_present(self):
         qc = QuantumCircuit(1)
         qc.x(0)
         qc.measure_all()
-        res = backend.run(qc, shots=1000).result().get_counts()
+        with warnings.catch_warnings():
+            with warnings.catch_warnings():
+                # TODO remove this catch once Aer stops using QobjDictField
+                warnings.filterwarnings("ignore", category=DeprecationWarning, module="qiskit")
+                res = backend.run(qc, shots=1000).result().get_counts()
```

Comment:
(I think it's no longer necessary and there was a strange nesting when catching warnings)

```suggestion
        res = backend.run(qc, shots=1000).result().get_counts()
```

Reply:
good catch!